forge-gui/src/main/java/forge/player/PlayerControllerHuman.java

```diff
@@ -2751,6 +2751,10 @@ private void addCardToZone(ZoneType zone, final boolean repeatLast, final boolea
                         forgeCard.changeToState(CardStateName.Flipped);
                         forgeCard.changeToState(CardStateName.Transformed);
                         forgeCard.changeToState(CardStateName.Modal);
+                        if (forgeCard.getCurrentStateName().equals(CardStateName.Transformed) ||
+                                forgeCard.getCurrentStateName().equals(CardStateName.Modal)) {
+                            forgeCard.setBackSide(true);
+                        }
```

Comment:
instead of using all the states, why not check forgeCard.getRules.getSplitType ?